pkg/session_recording/recordings.jsx

```diff
@@ -340,7 +349,7 @@
          * Set the cockpit location to point to the specified recording.
          */
         navigateToRecording(recording) {
-            cockpit.location.go([recording.id]);
+            cockpit.location.go([], $.extend(cockpit.location.options, { id: recording.id }));
```

Comment:
Yes, that's a good point. It would be useful to have the filtering options remembered and passed with the links.

However, can't we combine the two, perhaps? Have the ID as part of the "cockpit path", in the "fragment" part of the [URL](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax), and the filters in "cockpit options", in the "query" part?